knowledge-base/how-to-convert-svg-to-png.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Covert SVG graphics to PNG picture
+description: This article demonstrates how to display SVG graphics in PictureBox by converting it to PNG
+type: how-to
+page_title: How to convert SVG to PNG 
+slug: how-to-convert-svg-to-png
+position: 
+tags: 
+ticketid: 1400737
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product Version</td>
+		<td>13.0.19.222</td>
+	</tr>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+</table>
+
+
+## Description
+Currently, the [PictureBox](../report-items-picture-box) can hold only the formats supported by GDI+ (BMP, GIF, JPEG, EXIF, PNG, and TIFF). This KB article demonstrates how to convert SVG graphics to PNG picture through the examples below. Note that a  third party package- Svg should be added to the with NuGet packages.  
+
```

Comment:
correct "should be added to the with NuGet packages." for example to "should be added to the project References with NuGet packages."
Consider also adding a link to the external library so that it is easier for the user to find and refer it.